packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -227,14 +227,18 @@ name: foo
         ..createSync(recursive: true)
         ..writeAsStringSync('A');
       environment.buildDir.childFile('main.dart.js').createSync();
+      environment.buildDir.childFile('main.dart.js.info.json').createSync();
       environment.buildDir.childFile('main.dart.js.map').createSync();
       environment.buildDir.childFile('main.dart.js_1.part.js').createSync();
       environment.buildDir.childFile('main.dart.js_1.part.js.map').createSync();
 
-      await WebReleaseBundle(<WebCompilerConfig>[const JsCompilerConfig()]).build(environment);
+      await WebReleaseBundle(<WebCompilerConfig>[
+        const JsCompilerConfig(dumpInfo: true),
```

Comment:
Seemed...easier to just tweak the default test here.

happy to copy-paste if wanted